content/docs/tutorials/fullstack.md

```diff
@@ -35,20 +35,225 @@ In this tutorial, we will be going through the entire process of creating a full
-    - Twilio
-    - Send Grid
-- Payments
-    - Stripe
-- Unit Testing
-    - Pytest
 
+{{<break 1>}}
+
+## Create Cdev Project
+We will be starting this tutorial from the standard `quick-start` template.
+
+```bash
+cdev init diary-project --template quick-start
+```
+
+Now we can deploy our project to get a live Webhook
+
+```bash
+cdev deploy
+```
+This step should output the live url of your webhook and look like
+```
+Base API URL -> <your-endpoint>
+```
+
+{{<break>}}
+
+## Create Database Connection
+For this project we will be utilizing a relational database and the SqlAlchemy ORM.  An in-depth example of how to set up SqlAlchemy ORM can be found in the examples section of our website as a part of the ["Integrate RelationalDBs"](https://cdevframework.io/docs/examples/relationaldb/#sqlalchemy-orm)chapter. 
+
+Update your `src/hello_world/resources.py` file to: 
+{{<codesnippet `/source_code/diary_tutorial/step1_resource.py`>}}
+
+Now we can deploy our resources.
+```bash
+cdev deploy
+```
+
+{{<break 1>}}
+**Next, we are going to create our models.** Install `sqlalchemy_aurora_data_api`.
+
+```bash
+pip install sqlalchemy_aurora_data_api
+```
+Then, create a `src/hello_world/models.py` file and add the following code:
+
+{{<codesnippet `/source_code/diary_tutorial/models.py`>}}
+
+Now we are going to install and configure `alembic`.
+
+```bash
+pip install alembic
+```
+{{<tool_tip key="tip" summary="Alembic Library">}}
+`Alembic` is one of the best in class tools for working with SqlAlchemy. You can learn more about the tool from [their official documentation](https://alembic.sqlalchemy.org/en/latest/).
+{{</tool_tip>}}
+
+Initialize the needed files for `alembic` using the following command. We will need to edit some of the generated files to connect to our db.
+```bash
+alembic init src/alembic
+```
+Replace the code in your `src/alembic/env.py` with the following:
+{{<codesnippet `/source_code/diary_tutorial/alembic.py`>}}
+
+Now to make sure our values are registered in the `env.py` script, we need to set our environments. 
+
+```bash
+export SECRET_ARN=$(cdev output --value hello_world_comp.relationaldb.demo_db.secret_arn)
+```
+```bash
+export CLUSTER_ARN=$(cdev output --value hello_world_comp.relationaldb.demo_db.cluster_arn)
+```
+```bash
+export DB_NAME=<db_name>
+```
+{{<break 1>}}
+
+You should now be able to create automated migrations. 
+
+{{<tool_tip key="warning" summary="Auto Generation limits">}}
+You should familiarize yourself with the [limits of alembic auto generation](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect) and **ALWAYS** confirm the changes before applying them.
+{{</tool_tip>}}
+
+```bash
+alembic revision --autogenerate -m "Added entries table"
+```
+Apply the migration with the `upgrade` command. This will create our `Entry` Table.
+```bash
+alembic upgrade head
+```
+
+{{<break 1>}}
+Lets now connect to our DB and add an User.
```

Comment:
Add a sentence or two about it open an interactive session to the db that allows them to execute sql statements.